digiroad2-api-viite/src/main/scala/fi/liikennevirasto/digiroad2/ViiteApi.scala

```
@@ -308,23 +308,22 @@ class ViiteApi(val roadLinkService: RoadLinkService, val vVHClient: VVHClient,
   }
 
   put("/roadlinks/roadaddress/project/savenewroadlink") {
-    try { //check for validity
```

it didn't crash, but it did hide all error messages